tracing/src/main/java/io/micronaut/tracing/brave/BraveTracerFactory.java

```diff
@@ -94,7 +94,7 @@ SpanCustomizer spanCustomizer(Tracing tracing) {
     Tracer braveTracer(Tracing tracing) {
         BraveTracer braveTracer = BraveTracer.create(tracing);
         if (!GlobalTracer.isRegistered()) {
-            GlobalTracer.register(braveTracer);
+            GlobalTracer.registerIfAbsent(braveTracer);
```

Comment:
Shouldn't this be replacing the if statement?

Reply:
Done.